requirements-tests.txt

```diff
@@ -1,7 +1,5 @@
 -e git+https://github.com/ant31/pytest-sugar.git#egg=pytest-sugar
-backports.tempfile
 freezegun==0.3.12
-mockldap
```

Comment:
This overrides the `mockldap` included in requirements.txt. It would come from a different source.